docs/installation.md

```diff
@@ -34,10 +34,10 @@ Freqtrade provides a Linux/MacOS script to install all dependencies and help you
 ```bash
 git clone git@github.com:freqtrade/freqtrade.git
 cd freqtrade
-git checkout develop
 ./setup.sh --install
 ```
 
+    When cloning the repository the default working branch is name `develop`. This branch contains the last features (can be considered as relatively stable thanks to automated tests). The `master` branch contains the code of the last release (done once per month with a one week old snapshot of the `develop` branch to prevent packaging bugs so potentially more stable).
```

Comment:
I guess we should add rendered Note box on top of this?

```
!!! Note "Version considerations"
    When ...
```

(Note that the following indent MUST be 4 spaces to be rendered correctly (i fixed a few appearances with 2 spaces which were broken recently)

Reply:
oups forget the stage of this lines in this editor :-)

Reply:
I have take your title it is a good idea to not leave with default 'Note'